### PR TITLE
Update to alpha 3

### PR DESCRIPTION
**Pull request by razzeee**

### CODE REVIEW

**Comment by Goldenfreddy0703** [comment]
Hey sorry for bothering you but im using Kodi beta for my steam deck and was hoping we could update to alpha 3, is there any reason why these builds keep failing?

**Comment by razzeee** [comment]
The latest error means, that we will probably need to wait for https://github.com/xbmc/visualization.fishbmc/pull/55

**Comment by Goldenfreddy0703** [comment]
ahh, I noticed that. Thank you and hope we can update to alpha 3 soon.

**Comment by wsnipex** [comment]
> The latest error means, that we will probably need to wait for [xbmc/visualization.fishbmc#55](https://github.com/xbmc/visualization.fishbmc/pull/55)

we might just disable fishbmc until it's updated

**Comment by Goldenfreddy0703** [comment]
Ahh ok

**Comment by Goldenfreddy0703** [comment]
Niccceeeeeee I can start using kodi alpha 3 now, thank you!!!!

**Comment by razzeee** [comment]
What exactly is the feature your waiting for in the new alpha if I may ask?

**Comment by Goldenfreddy0703** [comment]
Hey, when installing, i get this warning. Should i be worried about that or is that ok sense the install is complete? I did test it and it is working. 
image

File: tv.kodi.Kodi.yml [hunk 1]
```diff
@@ -239,7 +234,7 @@ modules:
   - addons/vfs.libarchive/vfs.libarchive.json
   - addons/vfs.rar/vfs.rar.json
   - addons/vfs.sftp/vfs.sftp.json
-  - addons/visualization.fishbmc/visualization.fishbmc.json
+  # - addons/visualization.fishbmc/visualization.fishbmc.json
```
Comment (razzeee) [inline tv.kodi.Kodi.yml:237]: Add an issue to re-enable this later

**Comment by razzeee** [comment]
I'm not seeing that warning on fedora, not sure if it's a steamOS thing and not really in a mood test on my deck :)

**Comment by Goldenfreddy0703** [comment]
Oh then it could probably be a steam deck thing due to some limitations due to updates and all. That's ok after all, it still works and it has been the best so far.

**Review by wsnipex — APPROVED** [review]

**Pull request merged**